### DISCUSSION TITLE
1.2.15 stable

### DISCUSSION THREAD

**Discussion by trueromanus** [anilibria/anilibria-winmaclinux#166] [Анонсы]
- Исправлена кнопка перезапуска текущего видео при включенном проксировании
- Добавлена настройка Использовать надежное проксирование на странице TorrentStream
- Добавлена дата создания торрента в меню торрентов карточки релиза и общего списка
- Добавлена страница "Приложения" для скачивания и обновления связанных приложений
- Обновление прогресса загрузки при каждом заходе на страницу TorrentStream
- Добавлена кнопка "Обновить" в тултип окна для запуска инсталлятора

<hr /><em>This discussion was created from the release <a href='https://github.com/anilibria/anilibria-winmaclinux/releases/tag/1.2.15'>1.2.15 stable</a>.</em>

**Comment by ShadowMax** [comment]
> Добавлена страница "Приложения" для скачивания и обновления связанных приложений  

Я не совсем понял, речь идёт про этот пункт?
![image](https://github.com/anilibria/anilibria-winmaclinux/assets/10102994/85d0fe88-9248-4c7c-b389-8cca358aed0e)
Это вообще только заготовка на будущее? 
А если нет, то какую программу и откуда нужно сюда устанавливать?

**Reply by trueromanus** [reply]
> Я не совсем понял, речь идёт про этот пункт?

Речь идет про всю страницу с которой Вы вырезали кусок.

> Это вообще только заготовка на будущее?  А если нет, то какую программу и откуда нужно сюда устанавливать?

Идея этой страницы в том чтобы не надо было знать откуда устанавливать и обновлять дополнительные приложения а просто нажать на кнопку и "оно само". То что на скриншоте это приложение для установки и обновления приложения. Подробнее о нем можно почитать по следующим ссылкам:
https://t.me/desktopclientanilibria/529
https://t.me/desktopclientanilibria/564

**Comment by AndreyStudenttt** [comment]
есть баг, иза которого при выходе из плеера он открывается в маленькое окошке, что вызывает краш всего приложения
https://imgur.com/a/o5sS0co

**Reply by trueromanus** [reply]
Уточните какая система и какой плеер выбран у Вас?

**Reply by trueromanus** [reply]
Если видео превью не нужно можно отключить в настройках
![image](https://github.com/anilibria/anilibria-winmaclinux/assets/9040186/ca216a2d-9ef2-402b-a656-19cc64ebebd5)
